.github/workflows/build_ncs.yml

```diff
@@ -0,0 +1,64 @@
+name: Build binary for Nordic NCS
+
+# Runs when pushed to main, semver tagged release or any PR
+on:
+  push:
+    branches: [ main ]
+    tags: [ 'v*.*.*' ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    # Uses matrix to test combinations. Update when SDK changes or to add new boards.
+    strategy:
+      matrix:
+        ZEPHYR_SDK_VERSION: [0.16.1]
+        BOARD: ["nrf9160dk_nrf9160_ns","aludel_mini_v1_sparkfun9160_ns"]
+
+    steps:
+      - name: Install OS dependencies
+        run: >
+          sudo apt-get install -y
+          device-tree-compiler
+          git
+          ninja-build
+          python3
+          python3-pip
+          python3-wheel
```

Comment:
Is it better to use the OS version of `wheel` or `pip`?

Reply:
@cdwilson do you know? Something about OS versions not as up to date as PIP? Or does that not matter?

Reply:
TBH, I don't really know (I mainly use [pyenv](https://github.com/pyenv/pyenv) on Mac to avoid touching the system python at all).

I think in general some people might say to avoid using `pip` to install directly to the system python (vs. a virtualenv), and recommend using `apt` to install any python packages into the system python. But in this case, it probably won't make any difference unless Ubuntu is patching something to work with their system python.

Assuming that the `python3` package on Ubuntu doesn't bundle `pip3` along with it, you'll prob need to keep installing `python3-pip` via `apt` to make sure the `pip3` command is available. 

However, once you've got `pip3` installed, you can probably just `pip3 install wheel west` and get the latest versions from PyPI (unless you want to pin them to a specific version).